plugin-autoupdate-filter.php

```diff
@@ -3,7 +3,7 @@
 Plugin Name: Plugin Autoupdate Filter
 Plugin URI: https://github.com/a8cteam51/plugin-autoupdate-filter
 Description: Sets plugin automatic updates to always on, but only happen during specific days and times.
-Version: 1.4.1
+Version: 1.4.2
```

Comment:
Increasing the version number means we want to craft a new release, which will automatically push out to all of the sites that have this plugin installed (lots).

Does this set of changes match a version increase, as described by [SemVer](https://semver.org/)?

Reply:
🤔 I don't believe that it does. However, SemVer doesn't mention refactoring anywhere.

A quick search led me to [this discussion](https://github.com/semver/semver/issues/146). This argument makes sense to me:

> New bugs may have been introduced, so patch level must be updated

But in our case, if we don't update the patch level, the code won't actually be released until we make further changes to this plugin down the road.

So it seems like to me the question is, "Do we _want_ to release this refactor code and push it to all sites, or do we want to wait and have it included with the next release that merits a version increase per SemVer?"

I do think that we argue for a version bump on the basis plugin isn't updated frequently and doesn't have any kind of regular update schedule. Thoughts?

Reply:
Thanks for pointing at this discussion, that was a good read. I think in this case, let's not bump the version number or create a new release, and let these changes get rolled into the next version which contains one or more of these things:
- Bug fix
- Performance increase
- Feature or behavior change for the end user

Also noting here that I did load up this branch on a test site and verified that it works as intended without producing PHP errors in the log.

Reply:
@NickGreen - I went ahead and reverted the commit that contained the version bump, so we should be all set on this!